src/index.html

```diff
@@ -1,13 +1,51 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700;900&display=swap" rel="stylesheet">
+  </head>
+
+  <body>
+    <section class="product" data-qa="card">
+      <div class="product__photo">
+        <img
+          src="images/imac.jpeg"
+          alt="photo"
+          class="product__photo-img"
+        >
+      </div>
+      <h3 class="product__name">Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)</h3>
+      <p class="product__code">Код товара: 195434</p>
+      <div class="product__review">
+        <div class="product__stars stars--4">
```

Comment:
We still need "stars" block.
```suggestion
        <div class="product__stars stars stars--4">
```